src/main/resources/application.conf

```diff
@@ -2,7 +2,6 @@ akka {
   loggers = ["akka.event.slf4j.Slf4jLogger"]
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
   loglevel = "DEBUG"
-  stdout-loglevel = "OFF"
```

Comment:
Why was this removed?

Reply:
re-added, was trying to figure out how to get logs from remora app since logger.info was actually not actually logging but was something else.